DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -481,28 +516,11 @@ public IEnumerable<Course> GetCoursesEverUsedAtCentreByCategory(int centreId, in
             );
         }
 
-        public bool DoesCourseExistAtCentre(int customisationId, int centreId, int? categoryId)
-        {
-            return connection.ExecuteScalar<bool>(
-                @"SELECT CASE WHEN EXISTS (
-                        SELECT *
-                        FROM Customisations AS c
-                        JOIN Applications AS a on a.ApplicationID = c.ApplicationID
-                        WHERE CustomisationID = @customisationId
-                        AND c.CentreID = @centreId
-                        AND (a.CourseCategoryID = @categoryId OR @categoryId IS NULL)
-                    )
-                    THEN CAST(1 AS BIT)
-                    ELSE CAST(0 AS BIT) END",
-                new { customisationId, centreId, categoryId }
-            );
-        }
-

```

Comment:
Noticed this method was unused so I removed it. It's from another ticket I worked on when I made a new version of it and forgot to remove the old.